src/routes/github/create-branch/github-create-branch-get.test.ts

```diff
@@ -0,0 +1,22 @@
+import express, { Application } from "express";
+import { GithubCreateBranchGet } from "./github-create-branch-get";
+import supertest from "supertest";
+
+jest.mock("./github-create-branch-get");
```

Comment:
Why are you mocking the thing you're trying to test?

Reply:
It won't catch any issues in your route handler at all, making the test only testing if jest.mock works...

Reply:
The test case has been updated.